services/audit-service/src/main/java/datawave/microservice/audit/replay/util/LockableHazelcastCacheInspector.java

```diff
@@ -0,0 +1,91 @@
+package datawave.microservice.audit.replay.util;
```

Comment:
This doesn't seem audit-specific to me. Why isn't it in hazelcast-client?

Reply:
moved.